README.md

```diff
@@ -93,6 +93,22 @@ For example, for `Article` model:
 
 Your WebFlow collection `slug` should be `"articles"`.
```

Comment:
Can you please also update above:

> the output of `model_class.model_name.collection`

It's incorrect for models with multiple words, for example `JobListing`.

This should be correct:

> the output of `model_class.model_name.underscore.dasherize.pluralize`

Reply:
Added `to_s` to fix error:
```rb
> Article.model_name.underscore.dasherize.pluralize
# => NoMethodError: undefined method `underscore' for #<ActiveModel::Name:

> Article.model_name.to_s.underscore.dasherize.pluralize
# => "articles"
```

Added text:
For Rails models named with two words, make a collection that have a space between words in the name, WebFlow sets slug by replacing space for "-".

For example, for `FeaturedArticle` model:

```ruby
> FeaturedArticle.model_name.to_s.underscore.dasherize.pluralize
# => "featured-articles"
```
Your WebFlow collection `slug` in this case should be `"featured-articles"`.